control-plane/api-gateway/binding/validation.go

```diff
@@ -564,6 +564,51 @@ func externalRefsOnRouteAllExist(route *gwv1beta1.HTTPRoute, resources *common.R
 	return true
 }
 
+func checkIfReferencesMissingJWTProvider(filter gwv1beta1.HTTPRouteFilter, resources *common.ResourceMap, namespace string, invalidFilters map[string]struct{}) {
+	if filter.Type == gwv1beta1.HTTPRouteFilterExtensionRef {
+		externalFilter, ok := resources.GetExternalFilter(*filter.ExtensionRef, namespace)
+		if !ok {
+			return
+		}
+		authFilter, ok := externalFilter.(*v1alpha1.RouteAuthFilter)
+		if !ok {
+			return
+		}
+
+		for _, provider := range authFilter.Spec.JWT.Providers {
+			_, ok := resources.GetJWTProviderForGatewayJWTProvider(provider)
+			if !ok {
+				invalidFilters[fmt.Sprintf("%s/%s", authFilter.Name, namespace)] = struct{}{}
```

Comment:
We generally go `<namespace>/<resourcename>` since the namespace is the outer scope